CONTRIBUTING.md

```diff
@@ -1,238 +1,191 @@
-# Questions
+# Contributing
 
-This is the issue tracker for Ember Data. The Ember.js community uses this site
-to collect and track bugs and discussions of new features. If you are having
-difficulties using Ember Data or have a question about usage please ask a
-question on StackOverflow:
-[http://stackoverflow.com/questions/ask](http://stackoverflow.com/questions/ask?tags=ember.js&tags=ember-data)
-and tag your question with `ember.js` and `ember-data`.
+## Welcome!
 
-The Ember.js community is very active on StackOverflow and most questions
-receive attention the same day they're posted:
-http://stackoverflow.com/questions/tagged/ember.js
-http://stackoverflow.com/questions/tagged/ember-data
+We are so glad you are considering contributing to `ember-data`. Below you'll find sections
+detailing how to become involved to best ensure your contributions are successful!
 
-# Issues
+### Reporting Bugs
 
-Think you've found a bug or have a new feature to suggest? Let us know!
+Report issues you've discovered via the [issue tracker](https://github.com/emberjs/data/issues).
+We have provided an [issue template](./.github/ISSUE_TEMPLATE/bug.md) what will help guide you through the process.
+If you are unsure if something is a bug, the `#ember-data` channel on [Discord](https://discord.gg/zT3asNS) is
+a great place to ask for help!
 
-## Reporting a Bug
-1. Update to the most recent master release if possible. We may have already
-fixed your bug.
+### Discussion
 
-2. Search for similar issues. It's possible somebody has encountered
-this bug already.
+Before embarking on a fix, a new feature, or a refactor it is usually best to discuss the
```

Comment:
Does ember-data follow an RFC process or does the team permit (bigger) proposed changes via PR + conversation?

Reply:
We do, it is described just a short bit after this.